src/scss/components/optional/_cookie-warning.scss

```diff
@@ -0,0 +1,49 @@
+@mixin cookieWarning() {
+	.c-cookieWarning {
+	    background-color: $black;
+	    position: fixed;
+	    bottom: 0;
+		width: 100%;
+		$zIndex: zIndex(high);
+	}
+
+	.c-cookieWarning-inner {
+	    width: 940px;
+	    margin: 0 auto;
+	    padding: spacing() 0;
+	    overflow: hidden;
+
+    	@include media('<960px') {
+	        width: 620px;
+		}
+		@include media('<640px') {
+	        width: 300px;
+		}
+	}
+
+	.c-cookieWarning p {
```

Comment:
are we able to add a class to the cookie warning text? `.c-cookieWarning-content` or something similar

Reply:
The markup is in f-utils which would mean a PR & version there. Not sure its worth it?